requre/objects.py

```diff
@@ -69,14 +69,32 @@ def execute(cls, keys: list, func: Callable, *args, **kwargs) -> Any:
         """
         logger = logging.getLogger(cls.__name__)
         object_storage = cls(store_keys=keys)
-        if object_storage.persistent_storage.is_write_mode:
+
+        if object_storage.store_keys in object_storage.persistent_storage:
+            logger.debug(
+                f"{object_storage.store_keys} found in the persistent storage."
+            )
+
+        if (
+            object_storage.persistent_storage.mode == StorageMode.write
+            or (
+                object_storage.persistent_storage.mode == StorageMode.read_write
```

Comment:
Why is not supported read-write mode for ``DataTypes.List``? I think it does also sense in some cases. when you want to do it. But I understand that usecase is little bit not so straightforward as for Dict.
This "If" statement  is  little bit hard to understand